adapters/zmaticoo/zmaticoo.go

```diff
@@ -0,0 +1,164 @@
+package zmaticoo
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/prebid/openrtb/v19/openrtb2"
+	"github.com/prebid/prebid-server/v2/adapters"
+	"github.com/prebid/prebid-server/v2/config"
+	"github.com/prebid/prebid-server/v2/errortypes"
+	"github.com/prebid/prebid-server/v2/openrtb_ext"
+	"net/http"
+	"net/url"
+)
+
+var (
+	endPoint = "https://bid.zmaticoo.com/prebid/bid"
+)
```

Comment:
`endPoint` var is unused. I beleive this can be removed and we can rely on `config.Endpoint`